README.md

```diff
@@ -1,24 +1,61 @@
-# README
+# Rails and React Hello world project
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> Implement a connection between a Ruby on Rails back-end and React front-end - Two apps way.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+![](https://img.shields.io/badge/Github-blueviolet)
+![](https://img.shields.io/badge/Ruby_On_Rails-red)
+![](https://img.shields.io/badge/React-red)
 
-* System dependencies
 
-* Configuration
+### Install & Setup
 
-* Database creation
+To setup and install this sample ROR Blog project, follow the below steps:
+- Clone this project by the command: 
 
-* Database initialization
+```
+$ git clone https://github.com/teckim/hello-rails-back-end
+```
 
-* How to run the test suite
+- Then switch to the project folder by the bellow query:
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+$ cd hello-rails-back-end
+```
 
-* Deployment instructions
+- Install the required dependecies with `bundle install`
+- **Before running below commands update [config/database.yml](./config/database.yml) with your local `DB`credentials. Then run:**
+      - **`rake db:create`**
+      - **`rake db:migrate`**
+- With Ruby on Rails and PostgreSQL installed in your environment, you can run `rails server` to run the website.
```

Comment:
- [ ] **[OPTIONAL]** Since your frontend is set up to consume the API at port 8000, I would recommend you add the instruction for running the app for that port, since 3000 is the default port:

  ```
  rails s -p 8000
  ```